### PR TITLE
Fix issue #4857 - Add new entries

### PR DESCRIPTION
**Pull request by Heikrana**
Add new entries in various files in "data" folder.

### CODE REVIEW

**Comment by UKChris-osm** [comment]
Thanks @Heikrana 

Just out of interest, what are the edits in "_docs/src.6ebfba02.js _" as this file powers the NSI web site, but the file itself is built via the /app/ folder?

File: data/brands/shop/money_lender.json [hunk 1]
```diff
@@ -114,6 +114,19 @@
...
+    {
+      "displayName": "Crefisa",
+      "id": "crefisa-46cd18",
+      "locationSet": {"include": ["br"]},
+      "tags": {
+        "brand": "Crefisa",
+        "brand:website": "https://www.crefisa.com.br/",
+        "brand:wikidata": "Q10261396",
+        "brand:wikipedia": "pt:Crefisa",
+        "phone": "+55 86 3221 2476",
```
Comment (UKChris-osm) [inline data/brands/shop/money_lender.json:125]: Just an FYI, we also tend not to include contact details in this project, so you can also remove the phone number 👍

**Comment by Heikrana** [comment]
> Thanks @Heikrana
> 
> Just out of interest, what are the edits in "_docs/src.6ebfba02.js _" as this file powers the NSI web site, but the file itself is built via the /app/ folder?

It was automatically created after "npm run build", the first time I compiled the project.

File: data/brands/shop/money_lender.json (hunk 1 above)
Comment (Heikrana) [reply data/brands/shop/money_lender.json]: Okay, I have remove entry "phone" from the file "money_lender.json".

**Comment by UKChris-osm** [comment]
In that case I better wait for someone else (such as @bhousel) to check that out, as I'm not sure what "_npm run build_" has done to that file.

Thanks for this PR though @Heikrana 👍

**Comment by bhousel** [comment]
Thanks @Heikrana & @UKChris-osm  this looks ok..
It looks like some slightly different versions of dependencies, so I'll update everything to the latest and run the build again.

Also, I'm going to remove the `operator:phone=192`, that's another tag that we don't really add to this index.

**Pull request merged**